### PR TITLE
MTD-48: Add Translations button to right sidebar of node view.

### PR DESCRIPTION
**Pull request by kevwalsh**

### CODE REVIEW

File: config/sync/block.block.views_block__right_sidebar_latest_revision_block_1.yml [hunk 1]
```diff
@@ -26,6 +26,6 @@ settings:
 visibility:
   request_path:
     id: request_path
-    pages: "/node/*/edit\r\n/node/*/history\r\n/node/*/delete\r\n/node/*/clone\r\n/node/*/moderation-history\r\n"
+    pages: "/node/*/edit\r\n/node/*/history\r\n/node/*/delete\r\n/node/*/clone\r\n/node/*/moderation-history\r\n/node/*/translations\r\n"
```
Comment (kevwalsh) [inline config/sync/block.block.views_block__right_sidebar_latest_revision_block_1.yml:29]: This hides the right sidebar view from the node/nid/translations screen (much like it's hidden from the node/nid/edit screen).

File: docroot/themes/custom/vagovadmin/assets/scss/styles.scss [hunk 2]
```diff
@@ -76,8 +76,14 @@
 
 /* Right sidebar on node view pages. */
 
-.view-id-right_sidebar_latest_revision .views-row {
-  padding-bottom: 1em;
+.view-id-right_sidebar_latest_revision {
+  .translate-button {
+    padding-top: 1em;
+  }
+
```
Comment (kevwalsh) [inline docroot/themes/custom/vagovadmin/assets/scss/styles.scss:83]: Adds space between the Preview button and the Translate button.

File: config/sync/views.view.right_sidebar_latest_revision.yml [hunk 3]
```diff
@@ -393,6 +434,250 @@ display:
...
+          table: node_field_revision
+          field: nid
+          relationship: none
+          group_type: group
+          admin_label: ''
+          label: ''
+          exclude: false
+          alter:
+            alter_text: true
+            text: '<a href="/node/{{ nid__value }}/translations" class="button">Manage translation</a>'
```
Comment (kevwalsh) [inline config/sync/views.view.right_sidebar_latest_revision.yml]: This introduces inconsistency between "Translate" action on dashboard and "Manage translation" on right sidebar of node view.

File: config/sync/block.block.views_block__right_sidebar_latest_revision_block_1.yml (hunk 1 above)
Comment (swirtSJW) [reply config/sync/block.block.views_block__right_sidebar_latest_revision_block_1.yml:29]: Great idea to make it more similar.

File: docroot/themes/custom/vagovadmin/assets/scss/styles.scss (hunk 2 above)
Comment (swirtSJW) [reply docroot/themes/custom/vagovadmin/assets/scss/styles.scss:83]: Nice touch

**Review by swirtSJW — APPROVED** [review]
Look like great additions.

**Pull request merged**